README.md

```diff
@@ -48,6 +48,62 @@ cmake .. -D CMAKE_PREFIX_PATH=$CONDA_PREFIX -D CMAKE_INSTALL_PREFIX=$CONDA_PREFI
 make && make install
 ```
 
+## Installation within a mamba environment (wasm build instructions)
+
+First clone the repository, and move into that directory
+```bash
+git clone --depth=1 git@github.com:jupyter-xeus/xeus-r.git
+cd ./xeus-r
+```
+
+Now you'll want to create a clean mamba environment containing the tools you'll need to do a wasm build. This can be done by executing
+the following
+```bash
+micromamba create -f environment-wasm-build.yml -y
+micromamba activate xeus-r-wasm-build
+```
+
+You'll now want to make sure you're using emsdk version "3.1.45" and activate it. You can get this by executing the following
+```bash
+cd $HOME
+git clone https://github.com/emscripten-core/emsdk.git
+cd emsdk
+./emsdk install 3.1.45
+./emsdk activate 3.1.45
+source $HOME/emsdk/emsdk_env.sh
+```
+
+You are now in a position to build the xeus-r kernel. You build it by executing the following
+```bash
+micromamba create -f environment-wasm-host.yml --platform=emscripten-wasm32
```

Comment:
We should move the creation of the wasm environment to its own step and explain why we are creating a second environment and how this environment is different from the one we created above. Otherwise, I think it would be confusing for new users.